### PR TITLE
`perf.groovy` updates.

### PR DESCRIPTION
**Pull request by jorive**
- Increase how long performance artifacts are kept.
- Added an extra step to move files to be archived into a different folder.

### CODE REVIEW

**Comment by jorive** [comment]
@dotnet/rap-team PTAL

**Comment by jorive** [comment]
@dotnet-bot test ci please

File: perf.groovy [hunk 1]
```diff
@@ -639,13 +651,11 @@ parallel(
 
                     if (isPR) {
                         def opts = ""
-                        if (opt_level == 'min_opts')
-                        {
+                        if (opt_level == 'min_opts') {
```
Comment (brianrob) [inline perf.groovy]: Overall, the change looks fine to me.  One thing that I would like to know: Is there a style guide for *.groovy files?  I notice that much of your change is moving '{' characters onto the end of the previous line.  Is there clear guidance for this?

Adding @mmitche to comment.

Comment (jorive) [inline perf.groovy]: I haven't found any, but this file have them all mixed up. Assuming Java style, I reformatted the doc for consistency.

Comment (mmitche) [inline perf.groovy]: @brianrob Generally the same as java, which typically uses { on the same line as the if/loop/etc.

Comment (brianrob) [inline perf.groovy]: Ok, cool.  Thanks for the clarification.  Looks good.

File: perf.groovy [hunk 2]
```diff
@@ -101,27 +99,38 @@ def static getOSGroup(def os) {
...
                         }
 
                         if (isSmoketest) {
                             Utilities.setMachineAffinity(newJob, "Windows_NT", '20170427-elevated')
                         }
 
-                        // Save machinedata.json to /artifact/bin/ Jenkins dir
                         def archiveSettings = new ArchivalSettings()
-                archiveSettings.addFiles('bin/sandbox/Logs/Perf-*.*')
+                        archiveSettings.addFiles('bin/toArchive/')
```
Comment (adiaaida) [inline perf.groovy]: not 100% sure, but I think you need to include ** in this path, ie 'bin/toArchive/**'

Comment (jorive) [inline perf.groovy]: Thanks!

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Pull request merged**